README.md

```diff
@@ -11,19 +11,57 @@ These instructions will get you a copy of the project up and running on your loc
 
 ### Prerequisites
 
-Install Python 3 and pip
+To run the script (python or docker), you need to have an existing Google Agenda `credentials.json`.
+To obtain it, you'll have to create a google api account, add the Google Agenda API, then download the `credentials.json` file.
+See one of these links as an example documentation:
+
+- [Python Quickstart](https://developers.google.com/calendar/quickstart/python)
+- [Google API Authentication](https://flaviocopes.com/google-api-authentication/),
+
+Copy the `config-example.json` file to `config.json` and change it according to your needs (it should be documented enough for you to understand).
+Typically, you should have to change the used `dates` which contain the time slots for conferences.
+
+### Run with Docker
+
+Install Docker for your environment, then build the image with:
+
+```
+docker build . -t conference-hall-to-calendar
+```
+
+Finally, run the docker image by using the following command depending on your OS:
+
+On Mac/Linux:
+```sh
+docker run \
+  -v $PWD/my-config.json:/config.json \
+  -v $PWD/my-export.json:/export.json \
+  -v $PWD/credentials.json:/credentials.json \
```

Comment:
Not sure what would be the the best way to handle the credentials, because there is also the `token.pickle` file, but may not exists.

I see 2 solutions:
 - Declaring a `VOLUME ["auth"]` in the `Dockerfile` and adapt the code to read from this folder the `credentials.json` and `token.pickle`, then add a `--volume` option and doc for this folder
 - have only 1 volume in the docker command to mount a folder containing all the required files (export, credentials, token.pickle, ...), remove the options from the entrypoint and put them back in the command line (or add a default convention in `CMD` of `Dockerfile`)

Reply:
But that's what you told me as change request XD. I'm not able to refactor the python script (if it's necessary) and the goal of this issue is initiate the dockerfile. 
Just give me the preferable solution you want to use ^^

Reply:
Incremental building :stuck_out_tongue_winking_eye: 

For usability, I would go for a mix of the options.

For this : 

- In Dockerfile, move before execution to a folder that will be a volume containing the config files (as it's `PWD`, no need to change the python to load the credentials and token
   ```dockerfile
   [...]

   WORKDIR /conf
   ENTRYPOINT ["python", "/transformer.py",  "--input", "export.json", "--configuration", "config.json"]
   ```
- In README.md, document to use the exact names `export.json`, `config.json`and `credentials.json` and to put them in the same conf folder, then launch like this:
   ```shell
  docker run -v /absolute/path/to/conf/folder:/conf conference-hall-to-calendar
   ```

Reply:
Great :)

Reply:
and done :)